src/backend/distributed/planner/multi_master_planner.c

```diff
@@ -303,7 +304,7 @@ BuildAggregatePlan(Query *masterQuery, Plan *subPlan)
 		 * see nodeAgg.c:build_pertrans_for_aggref(). In that case we use
 		 * sorted agg strategy, otherwise we use hash strategy.
 		 */
-		if (!groupingIsHashable || hasDistinctAggregate)
+		if (!enable_hashagg || !groupingIsHashable || hasDistinctAggregate)
```

Comment:
@lithp / @mtuncer  will using this GUC cause trouble on windows?

Reply:
yes it causes build failure.  I do not think we should commit this change as it is.  
Is there a way to get this value by other means ? (spi etc)

Reply:
Hmm, that's interesting. Don't we use similar GUC variables all around the Citus? Do we use only Citus GUC variables?

Reply:
On second thought,  we can get this checked in. But we have to make sure that enable_hashagg is declared with PGDLLIMPORT. There are other variables declared with this in ```cost.h```